api.go

```diff
@@ -624,6 +632,29 @@ func (r *Raft) restoreSnapshot() error {
 	return nil
 }
 
+func (r *Raft) config() Config {
+	return r.conf.Load().(Config)
+}
+
+// ReloadConfig updates the configuration of a running raft node.
```

Comment:
```suggestion
// ReloadConfig updates the configuration of a running raft node. If the new config would be
// invalid it will return an error without changing the running node.
```